internal/pkg/manifest/http.go

```diff
@@ -47,37 +53,47 @@ func (r *RoutingRuleConfigOrBool) UnmarshalYAML(value *yaml.Node) error {
 	return nil
 }
 
-// RoutingRuleConfiguration holds the path to route requests to the service.
+// RoutingRuleConfiguration holds options for application load balancer.
 type RoutingRuleConfiguration struct {
+	Main                     RoutingRule   `yaml:",inline"`
+	TargetContainerCamelCase *string       `yaml:"targetContainer"` // "targetContainerCamelCase" for backwards compatibility
```

Comment:
Do we need to mark this as deprecated?
```suggestion
	TargetContainerCamelCase *string       `yaml:"targetContainer"` // Deprecated. kept for backwards compatibility
```